src/testcafe/schema.d.ts

```diff
@@ -2,7 +2,7 @@ export interface TestcafeBuilderOptions {
     allowMultipleWindows?: boolean;
     assertionTimeout?: number;
     browsers?: string[]; // default in schema.json
-    clientScripts?: string | string[]; //doesn't do anything yet
+    clientScripts?: string | string[];
```

Comment:
Please copy the typing from [here](https://github.com/DevExpress/testcafe/blob/cde211fd164938e0db0a29c7ede17fc473d6fdf1/ts-defs-src/runner-api/runner-api.d.ts#L146).